vector/build.gradle

```diff
@@ -365,6 +365,7 @@ dependencies {
     implementation "com.jakewharton.rxbinding3:rxbinding-appcompat:$rxbinding_version"
     implementation "com.jakewharton.rxbinding3:rxbinding-material:$rxbinding_version"
 
+//    implementation fileTree(include: ['*.jar', '*.aar'], dir: 'libs')
```

Comment:
we do not have `libs` dir can you revert this change please?

Reply:
Done